src/rules/no-large-snapshots.js

```diff
@@ -35,19 +89,32 @@ module.exports = {
     if (context.getFilename().endsWith('.snap')) {
       return {
         ExpressionStatement(node) {
-          reportOnViolation(context, node);
+          captureSnapshot(context, node);
         },
       };
     } else if (context.getFilename().endsWith('.js')) {
       return {
         CallExpression(node) {
-          const propertyName =
-            node.callee.property && node.callee.property.name;
-          if (
-            propertyName === 'toMatchInlineSnapshot' ||
-            propertyName === 'toThrowErrorMatchingInlineSnapshot'
-          ) {
-            reportOnViolation(context, node);
+          if (isTestCase(node) || isDescribe(node)) {
+            const [firstArgument] = node.arguments;
+            titles.push(getStringValue(firstArgument));
```

Comment:
Does this work for
```js
const x = 42;
test(`${x}`, () => {});
```
?

Reply:
good question, I will look for that and test it.

Reply:
@jeysal so the short answer is no.  The long answer is that i have a couple options.  One is to treat variables in a template literal as regular expressions when trying to resolve which snapshot they are associated with (not great). Another is to fallback to reporting the error in the snap for (also not great). Curious if you have any other thoughts.  Or if you know of a way to resolve the variables in the template literal so that the whole strong value can be reconstructed